src/graphql/resolvers/getOffersResolver.js

```diff
@@ -1,26 +1,11 @@
 import { TABLES } from "../environment";
-import { GetMany, GetSingle } from "./resolverHelper";
+import { GetMany, GetSingle, GetFiltered } from "./resolverHelper";
 import removeEmptyStrings from "../utils/removeEmptyStrings";
 import GetCompanyResolver from "./getCompanyResolver";
 
 // Gets all offers
-const GetOffersResolver = async (db, studentId = "") => {
-  let offers;
-  let offerParams = { TableName: TABLES.Offer };
-  if (studentId) {
-    offers = await db
-      .query({
-        ...offerParams,
-        KeyConditionExpression: "student_id = :id",
-        ExpressionAttributeValues: {
-          ":id": studentId
-        }
-      })
-      .promise();
-  } else {
-    offers = await db.scan(offerParams).promise();
-  }
-  offers = offers.Items || [];
+const GetOffersResolver = async (db, args) => {
+  let offers = await GetFiltered(db, TABLES.Offer, args.filters);
 
   let res = [];
   for await (let offer of offers) {
```

Comment:
Can we also add filtering to bonuses here? Better yet, is there a bonuses resolver that we can call?